setup/ci/travis.md

```diff
@@ -5,4 +5,38 @@ sidebar:
   nav: setup
 ---
 
-Documentation for this feature is coming soon! 
+{% include toc %}
+
+You can configure Spinnaker to use [Travis
+CI](https://travis-ci.org/){:target="\_blank"} as your Continuous Integration
+system, trigger pipelines with Travis, or add a Travis stage to a pipeline.
+
+## Prerequisites
+
+* You need a Travis user with an [API access token](https://docs.travis-ci.com/api/#authentication) so that you get only the
```

Comment:
If I understand the set-up properly, you really only have a Github user (who has granted some access scopes to Travis so it can kick off builds, etc.).  So this might be more clearly stated as a Github user with sufficient access to trigger builds (which I believe is just write permission to the repo).

Also "get only the repos you should see" suggests that without a token you'd see all repos, which I'm guessing isn't the case; maybe there's a way to rephrase.

Reply:
@gardleopard 
Are these understandings correct?